packages/testsuite/cypress/support/form-editing.ts

```diff
@@ -17,7 +17,8 @@ Cypress.Commands.add("cancelForm", (formId) => {
 
 Cypress.Commands.add("resetForm", (formId, managementApi, address) => {
   const resetButton = "#" + formId + ' a.clickable[data-operation="reset"';
-  cy.get(resetButton).click();
+  // the parameter "{ force: true }" is workaround for tests triggered in backgroud. For e2e testing is not required. 
```

Comment:
> the parameter "{ force: true }" is workaround for tests triggered in backgroud. For e2e testing is not required.

-> Setting the `force` parameter to `true` is a workaround turning off cypress checks whether the reset button is clickable as those checks sometimes fail. Next lines check the functionality (not) invoked by the button sufficiently.

Reply:
@OndrejKotek the `force: true` is not required anymore in this case. This should not be used at all.

The new fix is checking the server. In case it is WildFly then the edit button should not be visible so it will check the button doesn't exist. In case of EAP there is check to reset defaults values in case it is possible.

Please see update in the my first comment.